pipeline/README.md

```diff
@@ -13,6 +13,7 @@ log in to web console and install fly command.
     fly -t ci login -n dev
 
     fly -t ci set-pipeline -c pipeline-build.yml -p build
+    fly -t ci set-pipeline -c pipeline-analytics.yml -p analytics
```

Comment:
This pipeline no longer exists in this repository

Reply:
done